### DISCUSSION TITLE
Possibility to save diagrams with JavaScript

### DISCUSSION THREAD

**Discussion by JasonTysonParker** [jgraph/docker-drawio#152] [Q&A] [answered]
Hello,

I have deployed a Drawio container and was wondering if there is a way to save the current diagram using javascript or some sort of Drawio api?

Thank you very much in advance.

**Comment by davidjgraph** [comment, marked as answer]
There's no saving back-end implemented. It's designed to be saved to another file hosting app, either cloud storage or something like nextcloud.

**Reply by JasonTysonParker** [reply]
Thank you!!

Best Regards
